File: Codechef/Practice/Candy Store.cpp
https://www.codechef.com/problems/CANDYSTORE




#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin >> t;
	while(t--){
	    int x,y;
	    cin >> x >> y;
	    (y > x) ? cout << (y-x)*2 + x : cout << y;
	    cout << endl;
	}
	return 0;
}

File: Codechef/Practice/Candy Store.py
https://www.codechef.com/practice/course/basic-math-python/PPYBM01/problems/CANDYSTORE




t = int(input())

while t > 0:
    x, y = map(int, input().split())
    # Your code goes here
    if(y>x):
        print(2*y - x)
    else:
        print(y)
    t -= 1